quiz/small_theme.inc

```diff
@@ -11,10 +12,12 @@ function output_small_header($quiz)
     output_html_header('');
 
     $quiz_name = $quiz->short_name;
+    $user = User::load_current();
+    $logo = $user && $user->i_theme == "charcoal" ? "dp-logo-dark-360x68px.png" : "dp-logo-360x68px.png";
```

Comment:
I appreciate you working on this but I explicitly do not want to include or special-case things based on themes in the code. If we can figure out a way to do this purely in CSS I'm game, but we shouldn't do it in the code.

Reply:
@chrismiceli asked me to take a look at how the image looked, which I just kind of skimmed over when I first looked, but when I really started looking at it, the fuzziness of the white text really jumped out at me.

White on charcoal grey shows up flaws like that a lot more than black on light green/grey/blue does, though when I started looking at the PG theme, for contrast, I realized that even that looks fuzzy, especially when compared to the sharpness of the surrounding text.

All of this is leading up to -- I realize that the SVG file is significantly larger than the pngs, but can we get away with using an SVG logo?

Reply:
The SVG will be served up as gzip compressed so it'll be just 19k on the wire, and the browser will cache it, so the size shouldn't be an issue.

And just to be clear: I'm not suggesting that we must **theme** the SVG via CSS, but rather use CSS to serve up the appropriate file to begin with (as Chris mentioned in the task) rather than it being in PHP.

Reply:
using css to select the url is harder than I imagined since we don't know $code_url to generate an appropriate url

Reply:
You should be able to specify a relative URL against the final place of the CSS -- you don't need to know the absolute URL.

Reply:
Yes, you're right @cpeel, I was confused because I got the relative path wrong and it wasn't working /facepalm.